### PR TITLE
Feat : Add Landing Page

### LINKED ISSUE #1: Create A Landing Page

### PR DESCRIPTION
**Pull request by ZuberDunge**
image

### CODE REVIEW

**Review by siddhigate — APPROVED** [review]
Code is well structured. Looks good! You can convert images to webp format for better performance. Just a suggestion.
Good to merge👍

File: index.css [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+    font-size: 1.25rem;
+    background-color: var(--yellow);
+    color: var(--primary-color-dark);
+    padding: 0.5rem;
+    width: 100%;
+    font-weight: 500;
+}
+
+.featured-date {
+    font-size: 1.rem;
```
Comment (shivampandey0) [inline index.css]: `1.rem` is it a typo?

**Review by shivampandey0 — COMMENTED** [review]
Structured code with the use of variables.
- `important` is used (As Vikram said try to avoid it as much as possible)

File: index.css (hunk 1 above)
Comment (ZuberDunge) [reply index.css]: Yes, it was a typo. Thanks Shivam. Updated.

**Review by shivampandey0 — APPROVED** [review]
Fixed with Suggestions.
Good to merge

**Pull request merged**